src/metadata.js

```diff
@@ -93,7 +80,7 @@ export function getElementDetails(elementUris) {
 * @return {Array} An array of attribute objects
 */
 export function getAttributes(projectId) {
-    return get('/gdc/md/' + projectId + '/query/attributes').then(getIn('query.entries'));
+    return get('/gdc/md/' + projectId + '/query/attributes').then(r => r.ok ? r.json() : r).then(getIn('query.entries'));
```

Comment:
Seems `r => r.ok ? r.json() : r` is quite recurring here, maybe we could have something like `.then(getJSON)`